resources/views/welcome.blade.php

```diff
@@ -1,21 +1,11 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Mismatch Finder</title>
-</head>
-<body>
+<x-layout>
+    <p>Coming soon, to a screen near you...</p>
+-----------------------
     @auth
         <p>Hello, {{ Auth::user()->username }}! <a href="{{ route('logout') }}">Logout</a></p>
```

Comment:
I know these are ugly placeholders only, but I'd remove the Logout link here, just to indicate that it is now in the header

Reply:
This will be addressed in ticket [T286918](https://phabricator.wikimedia.org/T286918) since this PR only concernts the header and the tabs.